app/lib/actions/katello/repository/destroy.rb

```diff
@@ -36,7 +36,8 @@ def plan(repository, options = {})
 
         def finalize
           repository = ::Katello::Repository.find(input[:repository][:id])
-          delete_record(repository)
+          docker_cleanup = repository.content_type == "docker"
```

Comment:
Is there any advantage of passing this new options hash through rather than just checking the repo content_type in `delete_record` ?

Reply:
I wanted to have the check as an option for the method. Although we are not using this method anywhere else so we can do it either way. I will just take your suggestion.. Hash or not? :smile:

Reply:
Let's leave it! was curious if there was a specific purpose though